docs/modules/ROOT/pages/explanations/backup.adoc

```diff
@@ -0,0 +1,48 @@
+= Instance Backups
+
+This page explains the process and content of instance backups.
+
+
+// TODO: add link to appcat base docs regarding data protection.
+[IMPORTANT]
+====
+In general the `link here`[Data Protection] document applies and governs all aspects if not further defined below.
+Please read and understand this before reading on.
+====
+
+== Contents
+
+// Note: Maybe distinguish between Standalone and Replicated architectures, there may be differences (irrelevant for prototype)
+// There could also be different contents in different major versions.
+
+The backups contain the following files and directories:
+
+- `appendonly.aof`
+- `dump.rdb` (if existing)
+
+This is enough to restore Redis to a working state.
+
+== Location
+
+The backups are encrypted at rest in an S3 bucket.
+The bucket is residing on the same datacenter (region and provider) as the service instance is running on.
+
+== Retention policy
+
+The default retention policy applies.
+
+[IMPORTANT]
+====
+Deleting the service instance resource also irrevocably deletes all backups!
+====
+
+== Schedule
+
+The backups are triggered every 24h at a random time.
```

Comment:
This sounds like the time will change every 24h. Maybe specify it a bit:

```suggestion
The backups are triggered every 24h at a random time. The random time will be the same each day.
```

Reply:
If it were random every day then logically it can't be an interval of "every 24h", right? :smile: 
Going to change it to be more clear though.